README.md

```diff
@@ -241,20 +241,47 @@ publish.standard();
                 </p>
             </td>
         </tr>
+        <tr>
+            <td>
+                <code>checkFileTracking</code>
+            </td>
+            <td>
+                The CLI to execute which checks if there are any temporary changes.
```

Comment:
Probably want to mention that these were changes made as part of the publishing, for example the version number in the package.json file.